src/app/components/taskForm/cardFooter.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+
+type Props = {
+    submit: any;
+    taskId: number;
+    studentId: string;
+};
+
+export default class CardFooter extends React.Component<Props, any> {
+    constructor(props: Props) {
+        super(props);
+        this.state = {};
+    }
+
+    handleRepInput = (event: any) => {
+        this.setState({ assignmentRepo: event.target.value });
+    };
+
+    handleCommentInput = (event: any) => {
+        this.setState({ studentComment: event.target.value });
+    };
```

Comment:
Why do you write two almost equal functions? How about reusing some code? For example use arrow functions to close over local variables. See more details here https://reactjs.org/docs/faq-functions.html . Also you can use `name` instead of `data-letter` attribute.